### ISSUE TITLE
Error: Cannot read property 'style' of null

### ISSUE THREAD

**Issue by calebcordry** [ampproject/amphtml#22533]
About 12k errors in new canary on stackdriver. Sample stack trace: 

```
Error: Cannot read property 'style' of null
at style (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/src/style.js:119)
at (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/src/style.js:157)
at (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/extensions/amp-lightbox-gallery/0.1/swipe-to-dismiss.js:252)
at adjustForSwipePosition_ (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/extensions/amp-lightbox-gallery/0.1/swipe-to-dismiss.js:383)
at (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/src/service/resources-impl.js:1032)
at (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/src/service/vsync-impl.js:459)
at callTaskNoInline (https://raw.githubusercontent.com/ampproject/amphtml/1905222334000/src/service/vsync-impl.js:417)
```

Looks to be `amp-lightbox-gallery` related.
cc/ @ampproject/wg-ui-and-a11y

**Comment by calebcordry** [comment]
go/ampe/COuC3YCVi7zk7wE

**Comment by sparhami** [comment]
Looks like there was a spike on Sunday in the error rate, but has been lower before/after. I can't reproduce this when loading the canary version. I'm not 100% sure what is going on here. Here are something I know:

The error is happening here, with `swipeElement_` being null:

https://github.com/ampproject/amphtml/blob/f7d6f31ceeb1b937a68172f2e0dd9d71e5dc3d7d/extensions/amp-lightbox-gallery/0.1/swipe-to-dismiss.js#L252

This is set on the first touch (from touchstart) here:

https://github.com/ampproject/amphtml/blob/f7d6f31ceeb1b937a68172f2e0dd9d71e5dc3d7d/extensions/amp-lightbox-gallery/0.1/swipe-to-dismiss.js#L161

via:

https://github.com/ampproject/amphtml/blob/f7d6f31ceeb1b937a68172f2e0dd9d71e5dc3d7d/extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js#L645

At that point, `this.carousel_` cannit not be `null`, since it is initialized before the lightbox opens. I can't reproduce this error, 

I refactored this logic, but the data flow should be the same. The one thing I did change was I made the function use an object parameter instead of simple arguments, but I can't imagine that has any effect.

The minified code looks something like:

```javascript
                    a.Na.startSwipe({
                        swipeElement: a.I,
                        hiddenElement: c || b,
                        mask: a.aa,
                        overlay: a.T
                    })
```

and

```javascript
        U.prototype.startSwipe = function(a) {
            var b = this
              , c = a.swipeElement
              , d = a.hiddenElement
              , e = a.mask
              , f = a.overlay;
            this.oa = c;
            this.Ba = d;
            this.aa = e;
            this.ja = f;
            this.Ea(function() {
                Ab(b)
            })
        }
```

It is a little odd that the object properties are not being renamed, but it seems to be fine.

**Comment by sparhami** [comment]
Reopening until we can confirm whether or not we fixed this.

**Comment by jridgewell** [comment]
This is still not fixed, and it's by far the biggest error report in production.

**Comment by choumx** [comment]
As of writing, both this error and the attempted fix ("subsequent without first" error in #25375) are both present and firing at high volume in RTV 1911191835190. The latter is about half the volume of the first.

**Comment by Enriqe** [comment]
@ampproject/wg-ui-and-a11y Can you please reassign this issue? Seeing 8k occurrences in the current canary `2001251659540` in the last day.

**Comment by kristoferbaxter** [comment]
Assigning @wassgha to take a look.
